packages/swingset-theme-hashicorp/src/MDXComponents.tsx

```diff
@@ -1,14 +1,20 @@
 import { PropsTable } from './components/props-table'
 import { Heading } from './components/heading'
+import { LiveComponent } from './components/live-component'
+import { CodeBlock } from './components/code-block'
 
-export default {
+const MDXComponents = {
   h1: (props: any) => <Heading as="h1" {...props} />,
   h2: (props: any) => <Heading as="h2" {...props} />,
   h3: (props: any) => <Heading as="h3" {...props} />,
   h4: (props: any) => <Heading as="h4" {...props} />,
   h5: (props: any) => <Heading as="h5" {...props} />,
   h6: (props: any) => <Heading as="h6" {...props} />,
   p: (props: any) => <p className="ss-my-4" {...props} />,
-  pre: (props: any) => <pre className="ss-my-4" {...props} />,
+  pre: (props: any) => <CodeBlock {...props} />,
   PropsTable,
+  LiveComponent,
+  CodeBlock,
 }
+
+export default MDXComponents
```

Comment:
Exposes `Codeblock` and `LiveComponent` to `.mdx` 
Changes filename from `components` to `MDXComponents` to avoid confusion and be more semantic.